packages/shared/src/hooks/useOnPostClick.ts

```diff
@@ -67,10 +65,6 @@ export default function useOnPostClick({
                 null,
                 optional?.parent_id,
               ).extra,
-              feedback:
-                isFeedbackEnabled && post.type === PostType.Article
-                  ? true
-                  : undefined,
```

Comment:
@idoshamun I need to wait for [this](https://dailydotdev.slack.com/archives/C04UN615UAF/p1698388838610749) to resolve right?

Reply:
Yes for both comments, either we keep it for now or we keep this PR pending. Up to you